foia_hub/templates/request/form.html

```diff
@@ -17,7 +17,8 @@
 
 <section class="request form">
 
-  <form class="request" method="get" action="#">
+  <form class="request" method="POST" action="/request/noop/">
+    <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}"/>
```

Comment:
Oh, this is a remnant from when I was trying to get jinja2 to work. I should be able to use the tag now.